Patches/IsCardUnlocked.cs

```diff
@@ -1,14 +1,26 @@
 ﻿using HarmonyLib;
+using static System.Net.Mime.MediaTypeNames;
```

Comment:
This line can be safely removed, VS auto-added it when I was editing the code and I forgot to remove it.